.github/workflows/CI.yml

```diff
@@ -41,16 +42,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest]
-        libstdcxx-ng: [null, 12, 13]
-        exclude:
-          - os: macos-latest
-            libstdcxx-ng: 12
-          - os: macos-latest
-            libstdcxx-ng: 13
-          - os: ubuntu-latest
-            libstdcxx-ng: null
     runs-on: ${{ matrix.os }}
-    continue-on-error: ${{ ( matrix.libstdcxx-ng == 13 || matrix.os == 'macos-latest' ) }} # allow (expect) this to fail
```

Comment:
Does the Mac one now pass often enough that we want to require that it passes? I'd say if it still times out for Not Our Fault 20% of the time then maybe we want to keep the continue-on-error